assets/elm/Data/Board.elm

```diff
@@ -128,8 +155,13 @@ nextSnakeAndApples time snake apples =
 
 
 score : Board -> Int
-score { snake } =
-    List.length snake.body
+score { currentPlayerId, snakes } =
+    case Dict.get currentPlayerId snakes of
+        Nothing ->
+            0
```

Comment:
This actually needs refactoring, because this function shouldn't know anything about the way the snakes body is constructed.

Haskell has this nice function that is missing in Elm that does this, called (really annoyingly) `maybe`:

```
maybe :: b -> (a -> b) -> Maybe a -> b
```

... because it's a really common pattern to transform the innards of a maybe before returning it. I contemplated adding it into Utils, but I just hadn't got there yet.